content/edu/chicago.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: R-Ladies Chicago Package Development Workshop
+---
+
+## Women's Package Development Workshop by [R-Ladies Chicago](https://rladieschicago.org) and [FoRwards](https://forwards.github.io)
```

Comment:
Please use Forwards or R Forwards rather than FoRwards
```suggestion
## Women's Package Development Workshop by [R-Ladies Chicago](https://rladieschicago.org) and [Forwards](https://forwards.github.io)
```